src/lib/components/AssistantSettings.svelte

```diff
@@ -255,21 +283,126 @@
 				</div>
 				<p class="text-xs text-red-500">{getError("inputMessage1", form)}</p>
 			</label>
+			{#if $page.data.enableAssistantsRAG}
+				<div class="mb-4 flex flex-col flex-nowrap">
+					<span class="mt-2 text-smd font-semibold"
+						>Internet access <span
+							class="ml-1 rounded bg-gray-100 px-1 py-0.5 text-xxs font-normal text-gray-600"
+							>Experimental</span
+						>
+
+						{#if isHuggingChat}
+							<a
+								href="https://huggingface.co/spaces/huggingchat/chat-ui/discussions/385"
+								target="_blank"
+								class="ml-0.5 rounded bg-gray-100 px-1 py-0.5 text-xxs font-normal text-gray-700 underline decoration-gray-400"
+								>Give feedback</a
+							>
+						{/if}
+					</span>
+
+					<label class="mt-1">
+						<input
+							checked={!ragMode}
+							on:change={() => (ragMode = false)}
+							type="radio"
+							name="ragMode"
+							value={false}
+						/>
+						<span class="my-2 text-sm" class:font-semibold={!ragMode}> Disabled </span>
+						{#if !ragMode}
+							<span class="block text-xs text-gray-500">
+								Assistant won't look for information on the web and will be faster to answer.
```

Comment:
```suggestion
								Assistant won't look for information from Internet and will be faster to answer.
```

to be consistent with other use cases